### PR TITLE
Fix broken runtime in matplotlib. Need to include setuptools as runtime requirement for mpl_toolkits

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,11 +56,13 @@ requirements:
     - pyqt  # [not osx]
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
+    - setuptools  # [py35]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @ericdill do you know if only matplotlib on Python 3.5 uses `pkg_resources` or the issue does not manifest itself b/c for some reason only `conda-forge`'s does not have `setuptools`? If so we should remove the `[py35]` selector.

**Comment by ericdill** [comment]
> I am afraid that is b/c you are not getting the conda-forge Python package when Python <3.5.

Ohhhhhhhhhhhhh that makes a lot of sense.  Thanks!  Removing that selector now

**Comment by ocefpaf** [comment]
> Ohhhhhhhhhhhhh that makes a lot of sense. Thanks! Removing that selector now

Glad it makes sense to you b/c it does not make sense to me :smile:

I'd rather have `pip` and `setuptools` installed with `Python` as everywhere else :wink:

**Comment by ericdill** [comment]
> Glad it makes sense to you b/c it does not make sense to me :smile:

Ha-ha. I was simply referring to *what* the problem is! Not that the problem is inherently sensible :grinning: 

> I'd rather have pip and setuptools installed with Python as everywhere else :wink:

Yeah, I agree

**Comment by ocefpaf** [comment]
@ericdill can you bump the build number?

**Comment by ericdill** [comment]
@ocefpaf Ah, yes, that would be a good idea. Done in 89eaf73

**Comment by ocefpaf** [comment]
> @ocefpaf Ah, yes, that would be a good idea. Done in 89eaf73

Thanks @ericdill! My bad as I missed that in the first pass. Since we know this is passing I am merging this without waiting for the CIs so you can move your PR in staged-recipes forward.

**Comment by ericdill** [comment]
@ocefpaf sounds good to me. Thanks for moving quickly on this

**Comment by ocefpaf** [comment]
Feel free to ping me multiple times and/or calling me over gitter. Due to the noise in `conda-forge` it is easy to miss stuff.

**Comment by ericdill** [comment]
> Feel free to ping me multiple times and/or calling me over gitter. Due to the noise in conda-forge it is easy to miss stuff.

I'll try not to abuse this permission-to-harass-you you've just given me :grinning:

**Comment by ericdill** [comment]
@ocefpaf I should probably add this same change into the devel branch, right?

**Comment by ocefpaf** [comment]
Yes please!

**Comment by ericdill** [comment]
@ocefpaf #51

**Pull request merged**